RickLeinecker/Fortuna
# Api jwt verification

**Description**
Created a route to specifically verify jwt tokens and added auth to updateCassus route.
Resolves #173 #157 
**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
Call the /verify route on Postman with a x-auth-token header received from login:
* Header key: `x-auth-token`
* Header value: Token received on login. No quotes.

See if it returns a success response. 
Also call the /updateCasus route and see if the auth gives feedback.

**Final Checklist:**
Go down the list and check them off.
